src/Exception/RequestException.php

```diff
@@ -24,10 +24,27 @@ class RequestException extends \Exception implements RequestExceptionInterface
      */
     public function __construct($response)
     {
-        $error = $response['error'];
-        $this->responseCode = $error['code'];
-        $this->message = $error['message'];
-        $this->type = $error['type'];
+        $message = '';
+        $code = null;
+        $type = null;
+        if (is_string($response)) {
```

Comment:
Please look at [Scrutinizer](https://scrutinizer-ci.com/g/solutionDrive/HipchatAPIv2Client/inspections/8ee45def-bd36-4ebe-9dbd-76f8c9cd0eaa/issues/files/src/Exception/RequestException.php?status=new&orderField=path&order=asc&honorSelectedPaths=0) to find a new issue because the docstring stated that response is always a string and cannot be an array! It was previously an error because it was a string in docstring and used as array in the method, but can you please check that issue and change the docstring?